astropy/coordinates/tests/test_representation.py

```diff
@@ -1046,6 +1049,16 @@ class LogDRepresentation(BaseRepresentation):
                                         ('lat', Latitude),
                                         ('logr', u.Dex)])
 
+    # now try passing in a differential
+    diff = CartesianDifferential(d_x=1 * u.km/u.s,
```

Comment:
Oh, I see now a possible issue in the `scale_factors`. Anyways, if you want to try your own version off of master I can just rebase on that